server/api/api.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const Sighting = require('../models/Sighting.js');
+const sightingsController = require('../lib/sightingsController.js');
+
+const router = express.Router();
+
+// TODO: Adjust API to return data or err instead of unnamed JSON
+async function getSightings(req, res) {
+	try {
+		const result = await sightingsController.getSightings();
+		console.log('got sightings!');
+		res.status(200).json(result);
+	} catch (err) {
+		console.log(err);
+		res.status(500).json(err);
+	}
+}
+
+router.get('/sighting', getSightings);
+
+async function addSighting(req, res) {
+	const newSight = new Sighting(req.body);
```

Comment:
I'd move this into `sightingsController.addSighting`

Reply:
removed line 22 and now call sightingsController.addSighting(req.body)